eventhandlers.go

```diff
@@ -143,7 +147,11 @@ func HandleActionTriggeredEvent(myKeptn *keptn.Keptn, incomingEvent cloudevents.
 		actionResult.Result = "pass"
 		actionResult.Status = "succeeded"
 
-		myKeptn.SendActionFinishedEvent(&incomingEvent, actionResult, data.Labels, incomingEvent.Source())
+		myKeptn.SendActionFinishedEvent(&incomingEvent, actionResult, data.Labels, "ansibletower-service")
+		if err != nil {
+			log.Printf("Got Error From SendActionFinished: %s", err.Error())
```

Comment:
```suggestion
			log.Printf("Got Error From SendActionFinishedEvent: %s", err.Error())
```
Just to keep the standard, I added "Event" here.

Reply:
good catch - fixed it now